% -*-latex-*-
%\documentclass[pdftex,handout]{beamer}
\documentclass[pdftex,12pt]{beamer}
\usepackage{etex}

\let\latexput\put
\usepackage{pictex}
\let\pictexput\put
\let\put\latexput

\usepackage{color}
\setbeamercovered{transparent}
\begin{document}
\title{Archaic Genes in Modern Humans}
\author{Alan R. Rogers}

\frame{\titlepage}

\begin{frame}
\frametitle{Outline}
\begin{itemize}
\item History of thought about archaic admixture
\item Inferring admixture from derived allele patterns and aDNA.
\item Inferring admixture from LD in modern DNA.
\item Multiple Denisovan populations  
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Hypotheses of modern human origins}
\begin{description}
\item[Multiregional]
Modern humans evolved in a worldwide population united by gene flow
(Wolpoff 1989).
\item[Replacement]
``A single origin, outward migration of separate stirps, like the sons
  of Noah, and an empty world to occupy, with no significant threat of
  adulteration by other gene pools or even evaporating gene puddles''
  (Howells 1976).
\item[Admixture]
  Expansion from a single origin, involving ``encounters between
  populations of modern man and of other forms, with consequent gene
  flow''   (Howells 1976; also: Brauer 1984, Smith et al 1989,
  Trunkhaus 2005).
\end{description}
\end{frame}

\begin{frame}
\frametitle{Geneticists on admixture}
Consensus among geneticists during 1990s: little or no admixture
between moderns and archaics.

\bigskip
Eswaran et al 2002, 2005: argued for admixture

\bigskip

Hawks et al 2006: Small amounts of admixture can have a big effect,
because advantageous alleles are most likely to introgress.

\bigskip

Green et al 2010: sequenced Neanderthal genome; showed that 1.3--2.7\% of
European genes came from Neanderthals. [current estimate 1.5--2.1\%]
\end{frame}

\begin{frame}
\frametitle{Outline}
\begin{itemize}
\item[$\circ$] History of thought about archaic admixture
\item Inferring admixture from derived allele patterns and aDNA.
\item Inferring admixture from LD in modern DNA.
\item Multiple Denisovan populations  
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Nucleotide site patterns}

\begin{columns}
\column{0.5\textwidth}
{\centering
\begin{tabular}{l@{\hspace{0.4em}}c@{\hspace{0.4em}}c@{\hspace{0.4em}}c}
&\multicolumn{3}{c}{Nucleotide}\\
&\multicolumn{3}{c}{Site Pattern}\\
 &\emph{xy}&\emph{yn}&\emph{xn}\\
\cline{2-4}
Afr (X) & 1 & 0 & 1\\
Eur (Y) & 1 & 1 & 0\\
Nea (N) & 0 & 1 & 1\\
Chi & 0 & 0 & 0\\
\hline
\#  &303,340&103,612&95,347
\end{tabular}\\}
\column{0.5\textwidth}

Ancestral allele (0) is shared with chimp.

\bigskip
Mutant allele (1) shared by two human populations.

\bigskip
Pattern \emph{xy}: most common; reflects history of population splits.

\bigskip
Patterns \emph{yn} \& \emph{xn}: how do they arise?

\bigskip
Why does \emph{yn} exceed \emph{xn}?
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Population tree}
\centering\input{figpoptree}\\[1ex]
$X$, Africa; $Y$, Europe; $N$, Neanderthal; $C$, chimpanzee\\
\end{frame}

\begin{frame}
\frametitle{Embedded gene genealogy with mutation}
\begin{columns}
\column{0.5\textwidth}
\centering\input{figcongruent}\\[1ex]
\column{0.5\textwidth}
\begin{itemize}
\item Genealogy of 4 genes shown in color.
\item Bullet ($\bullet$) marks mutation from allele~0 to allele~1.
\item Descendants of mutant have allele~1; others have 0.
\item Gene genealogy matches phylogeny
\item Mutant allele shared by closest relatives, $X$ and $Y$.
\end{itemize}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Incomplete lineage sorting}
\centering\input{figlinsort}\\[1ex]
These two should be equally common\\
\end{frame}

\begin{frame}
\frametitle{Nucleotide site patterns again}

\begin{center}
\begin{tabular}{lccc}
&\multicolumn{3}{c}{Nucleotide Site Pattern}\\
 &\emph{xy}&\emph{yn}&\emph{xn}\\
\cline{2-4}
African  (X)   & 1 & 0 & 1\\
European (Y)   & 1 & 1 & 0\\
Neanderthal (N) & 0 & 1 & 1\\
Chimpanzee  & 0 & 0 & 0\\
\hline
\# sites    &303,340&103,612&95,347
\end{tabular}
\end{center}

\bigskip
Common pattern (\emph{xy}) reflects history of population splits.

\bigskip
Absent admixture, the other two should be equally common

\bigskip
Why does \emph{yn} exceed \emph{xn}?
\end{frame}

\begin{frame}
\frametitle{Neanderthal admixture inflates \emph{yn} site pattern}
{\centering\input{figmix}\\[2ex]
\small Key: $X$, Africa; $Y$, Europe; $N$, Neanderthal; $C$, chimpanzee.\\}
\end{frame}

\begin{frame}
\frametitle{Estimate from Neandertal DNA}
\begin{itemize}
\item DNA of modern Eurasians is 1.5--2.1\% Neandertal\\
\hfill  {\footnotesize(Pr{\"u}fer et al 2014)}.

\item Same is true for modern people of east Asia and Papua New
  Guinea, but not Africa. \hfill{\footnotesize Green et al (2010)}

\item Admixture must have occurred \emph{after} moderns left Africa
  but \emph{before} they expanded throughout the world.

\item Eurasian introgressed segments most similar to Neanderthal from
  Caucasus.\hfill   {\footnotesize(Pr{\"u}fer et al 2014)}
\end{itemize}
\end{frame}

%\begin{frame}
%\frametitle{Eurasians share some derived alleles with archaics}
%
%Neanderthal matches French 4.6\% more often than Yoruban (African).
%
%\bigskip
%
%Denisova matches French 1.8\% more often than Yoruban (African).
%
%\bigskip
%
%Archaic component of Eurasian genome more Neanderthal than Denisovan.
%\par\hfill\small Green et al 2010; Reich et al. 2010.
%\end{frame}

%\begin{frame}
%\frametitle{So do Asians and Papuans}
%
%Asians and Papuans carry as many Neanderthal alleles as Europeans do:
%1.3--2.7\%.
%\end{frame}

%\begin{frame}
%\frametitle{Segments of Neanderthal chromosome}
%\begin{itemize}
%\item low variation w/i humans
%\item more common in Europeans
%\item similar to Neanderthal
%\item large difference from African
%\end{itemize}
%\end{frame}

%\begin{frame}
%\frametitle{PCA plot based on genomes of chimp, neanderthal, and
%  denisova}
%\includegraphics[height=0.8\textheight]{pca1.pdf}
%\end{frame}

%\begin{frame}
%\frametitle{Zooming in on human portion of plot}
%\includegraphics[height=0.8\textheight]{pca2.pdf}
%\end{frame}

%\begin{frame}
%\frametitle{Entire human population rather than individual specimens}
%\includegraphics[height=0.8\textheight]{pca3.pdf}
%\end{frame}

%\begin{frame}
%\centering
%\includegraphics[height=0.85\textheight]{reichtree.pdf}\\
%\end{frame}

\begin{frame}
\frametitle{Denisovan DNA most common in Australia, NG, and Oceania}
\centering
\includegraphics[width=\textwidth]{denisovamap.pdf}\\
\end{frame}

\begin{frame}
  \frametitle{Some estimators have large biases}
  \begin{columns}
    \column{0.6\textwidth}
    \input{figrn_kappa}
    \column{0.4\textwidth}
    \raggedleft
    Bias is pronounced in populations that received gene flow from
    Denisovans as well as Neanderthals.

    \bigskip

    Worse in East Asia than in Europe.

    \bigskip

    Worse still in Melanesia.

    \bigskip
    All of the estimates reviewed above are suspect.
  \end{columns}
\end{frame}

\begin{frame}
  \frametitle{Legofit} Our own estimator, Legofit, is an estimator
  that avoids this bias. I'll discuss it in a separate lecture.
\end{frame}  

\begin{frame}
\frametitle{Outline}
\begin{itemize}
\item[$\circ$] History of thought about archaic admixture
\item[$\circ$] Inferring admixture from derived allele patterns and aDNA.
\item Inferring admixture from LD in modern DNA.
\item Multiple Denisovan populations  
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Using LD to discover admixed chromosome segments}
\begin{enumerate}
\item Recent introgression $\rightarrow$ modern genomes should contain
  long segments archaic chromosome.
\item These segments should differ a lot, because of the long
  separation time.
\end{enumerate}
\end{frame}

\begin{frame}
\frametitle{Method of Vernot and Akey (2014)}
\begin{enumerate}
\item $S^*$ statistic {\footnotesize (Plagnol \& Wall 2006)}
    uses modern LD to find candidate introgressed segments.
\item Accept candidate if matches Neanderthal sequence better than
  chance.
\item Studied 379 Europeans and 286 East Asians.
\item Found 15~Gb introgressed sequence spanning 20\% of Neanderthal
  genome.
\end{enumerate}
\end{frame}

\begin{frame}
\frametitle{Neandertal segments in modern genomes}
\begin{columns}
\column{0.5\textwidth}
\includegraphics[height=0.8\textheight]{Vernot-Akey.png}\\
\column{0.5\textwidth}
\raggedleft
\textcolor{red}{red}: Asians\\

\bigskip

\textcolor{blue}{blue} Europeans\\

\bigskip

Note the large ``deserts'' that largely lack archaic DNA. Suggests
selection against archaic alleles.

\bigskip

\mbox{}\hfill\footnotesize(Vernot and Akey 2014)\\
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Asians seem to have more Neandertal than Europeans---why?}
{\centering\includegraphics[height=0.8\textheight]{Vernot-violin.png}\\[-4ex]}
\mbox{}\hfill{\small Vernot and Akey (2014)\\}
\end{frame}

\begin{frame}
  \frametitle{Asian excess may be misleading}
  \includegraphics[width=\textwidth]{Macia21-map.png}\\

  Neanderthal fragments are longer in East Asia. Long fragments are easier
  to recognize. May explain East Asian excess. {\footnotesize (Macia et al
    (2021))}\\ 
\end{frame}

\begin{frame}
  \frametitle{Why are Neanderthal fragments short in Europe?}
  Perhaps admixture was older in Europe?  Macia et al (2021) say no:
  the genomic regions covered by Neanderthal fragments are the same
  Europe and Asia. Implies a single episode of admixture.

  \bigskip

  More derived alleles in Europe than Asia $\Rightarrow$ Shorter
  generation time in Europe $\Rightarrow$ More recombination in
  Europe.

  \bigskip

  So apparent excess of Neanderthal DNA in Asia may be an artifact
  caused by longer generation time there.
\end{frame}

\begin{frame}
\frametitle{Outline}
\begin{itemize}
\item[$\circ$] History of thought about archaic admixture
\item[$\circ$] Inferring admixture from derived allele patterns and aDNA.
\item[$\circ$] Inferring admixture from LD in modern DNA.
\item Multiple Denisovan populations  
\end{itemize}
\end{frame}

\begin{frame}
  \frametitle{Papuans got DNA from 2 Denisovan pops}
  \begin{columns}
    \column{0.5\textwidth}
    \includegraphics[width=\linewidth]{jacobs-papua.png}
    \column{0.5\textwidth}
    \raggedleft
    Vertical axis: length of introgressed segment

    \bigskip

    Horizontal: diff btw segment and Denisovan genome as fraction of
    Denisovan-modern diff.

    \bigskip

    Two Denisovan populations: one 0.15 and one at 0.2.

    \bigskip

    Jacobs et al (2019)
  \end{columns}
\end{frame}  

\begin{frame}
  \frametitle{E Asians got DNA from 1 Denisovan pop}
  \begin{columns}
    \column{0.5\textwidth}
    \includegraphics[width=\linewidth]{jacobs-easia.png}
    \column{0.5\textwidth}
    \raggedleft
    Vertical axis: length of introgressed segment

    \bigskip

    Horizontal: diff btw segment and Denisovan genome as fraction of
    Denisovan-modern diff.

    \bigskip

    One Denisovan population at 0.07.
  \end{columns}
\end{frame}  

\begin{frame}
\frametitle{Summary}
\begin{itemize}
\item Neanderthal: all non-African populations have about same level
  (1.5--2.1\%) of admixture. Suggests that admixture happened in one region.
\item Denisovan: highest levels found in populations far from
  Africa. Suggests that admixture accumulated gradually.
\item We can recognize chromosomal segments derived from Neanderthals
  or Denisovans, because they differ from modern haplotypes but
  resemble archaic haplotypes.
\item Large regions of Eurasian genomes have little or no archaic
  introgression.
\item East Asians have more archaic DNA than Europeans do.  
\item Papuans got DNA from 2 Denisovan pops.
\item E Asians got DNA from a 3rd.
\end{itemize}
\end{frame}

\end{document}
