% -*-latex-*-
%\documentclass[handout]{beamer}
\documentclass[]{beamer}
\usepackage{etex}
\usepackage{color}
\setbeamercovered{transparent}
\begin{document}
\title{Selection on Archaic Genes}
\author{Alan R. Rogers}

\frame{\titlepage}

\begin{frame}
\frametitle{Outline}
\begin{itemize}
\item Adaptive introgression of archaic alleles.
\item Purifying selection against archaic alleles.
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Using LD to discover admixed chromosome segments}
\begin{enumerate}
\item Recent introgression $\rightarrow$ modern genomes should contain
  long segments archaic chromosome.
\item These segments should differ a lot, because of the long
  separation time.
\end{enumerate}
\end{frame}

\begin{frame}
\frametitle{OAS1 innate immunity locus}
\begin{itemize}
\item two forms of gene in Melanesia:
\item one shared with rest of world
\item one only in Melanesia
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Worldwide frequency of Melanesian OAS1 allele}
{\centering\includegraphics[width=\textwidth]{OAS1-worldmap.png}\\}
\end{frame}

\begin{frame}
\frametitle{Melanesian OAS1 allele w/i Melanesia}
{\centering\includegraphics[width=\textwidth]{OAS1-melanesia.png}\\}
\end{frame}

\begin{frame}
\frametitle{Melanesian OAS1 allele is old yet young}
\begin{itemize}
\item The 2 alleles differ at many nucleotide sites $\Rightarrow$
  separation time ${\sim}3.4$~my.
\item Long (90~kb) LD block $\Rightarrow$ they've been together only
  ${\sim}25$~ky
\item Melanesian allele matches that in Denisovan hominin skeleton.
\item[$\Rightarrow$] archaic admixture into Melanesia
\end{itemize}
\end{frame}

\begin{frame}
  \frametitle{HLA loci}

  The loci of the HLA system underlie adaptive immunity in humans.

\bigskip

Abi-Rached et al (2011) estimate that $>50$\% of Eurasian HLA alleles
came from archaics.
\end{frame}

\begin{frame}
\frametitle{Outline}
\begin{itemize}
\item[$\circ$] Adaptive introgression of archaic alleles.
\item Purifying selection against archaic alleles.
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Method of Vernot and Akey (2014)}
\begin{enumerate}
\item $S^*$ statistic {\footnotesize (Plagnol \& Wall 2006)}
    uses modern LD to find candidate introgressed segments.
\item Accept candidate if matches Neanderthal sequence better than
  chance.
\item Studied 379 Europeans and 286 East Asians.
\item Found 15~Gb introgressed sequence spanning 20\% of Neanderthal
  genome.
\end{enumerate}
\end{frame}

\begin{frame}
\frametitle{Neandertal segments in modern genomes}
\begin{columns}
\column{0.5\textwidth}
\includegraphics[height=0.8\textheight]{Vernot-Akey.png}\\
\column{0.5\textwidth}
\raggedleft
\textcolor{red}{red}: Asians\\

\bigskip

\textcolor{blue}{blue} Europeans\\

\bigskip

Note the large ``deserts'' that largely lack archaic DNA. Suggests
selection against archaic alleles.

\bigskip

\mbox{}\hfill\footnotesize(Vernot and Akey 2014)\\
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Barrier to gene flow between Neanderthals and moderns}
\begin{columns}
\column{0.5\textwidth}
\includegraphics[width=\linewidth]{Vernot-Akey-barrier.png}
\column{0.5\textwidth}
Introgressed segments rare in genomic regions where moderns differ
greatly from Neanderthals.

\bigskip

Suggests partial reproductive isolation.
\end{columns}
\end{frame}

%\begin{frame}
%\begin{columns}
%\column{0.7\textwidth}
%\centering
%\includegraphics[width=\linewidth]{Sankararaman-Bstat.png}\\
%{\footnotesize Decreasing selection $\rightarrow$\\}
%\column{0.3\textwidth}
%\textcolor{blue}{Neanderthal inserts rare where selection is strong}
%
%\bigskip
%
%\mbox{}\hfill{\small Sankararaman et al (2014)\\}
%\end{columns}
%\end{frame}

\begin{frame}
\frametitle{History of population size from single diploid genomes:
  the pairwise sequentially Markovian coalescent (PSMC)}
\includegraphics[width=\linewidth]{Meyer-pophist.png}\\
\mbox{}\hfill\footnotesize(Meyer et al 2012)\\
\end{frame}

\begin{frame}
\frametitle{PSMC with Neanderthal as well as Denisova}
\includegraphics[width=\linewidth]{Prufer-pophist.jpg}\\
\mbox{}\hfill\footnotesize(Pr{\"u}fer et al 2014)\\
\end{frame}

\begin{frame}
\frametitle{Estimated times to most recent common ancestor (TMRCA)}
\includegraphics[width=\linewidth]{Prufer-tmrca.png}\\[2ex]

Long stretches of low TMRCA $\Rightarrow$ recent close inbreeding.\\
\mbox{}\hfill {\small (Pr{\"u}fer et al 2014)\\}
\end{frame}

\begin{frame}
\frametitle{Altai Neanderthal was very closely inbred}
\includegraphics[width=\linewidth]{Prufer-pedigree.png}\\[1ex]

All of these pedigrees are plausible.\\
\mbox{}\hfill {\small (Pr{\"u}fer et al 2014)\\}
\end{frame}

\begin{frame}
\frametitle{Long history of small population size}
\includegraphics[width=\linewidth]{Prufer-Hruns.png}\\[2ex]

Even after removing the effects of inbreeding during the last few
generations, the Altai Neandertal is still highly inbred.
\end{frame}

\begin{frame}
\frametitle{Coding sequence variation in archaics}

Castellano et al (2014) study 17,367 protein-coding genes in several
Neanderthals, the Denisovan, and several moderns.
\end{frame}

\begin{frame}
\frametitle{Neanderthals had low heterozygosity}
{\centering
\begin{tabular}{llr}
        &            & Heterozygosity\\
Species & Population &  $\times 1000$\\
\hline
Neanderthal     & El Sidr{\'o}n & 0.143\\
                & Vindija       & 0.127\\
                & Altai         & 0.113\\[1ex]
Modern          & African       & 0.507\\
                & European      & 0.387\\
                & Asian         & 0.358
\end{tabular}\\[2ex]}

Low heterozygosity $\Rightarrow$ small population.

\bigskip

Long runs of homozygosity $\Rightarrow$ recent close inbreeding.\\
\mbox{}\hfill {\small Castellano et al (2014)\\}
\end{frame}

\begin{frame}
\frametitle{Neanderthals had low heterozygosity}
{\centering\includegraphics[height=0.8\textheight]{hetarch.png}\\}
\end{frame}

\begin{frame}
\frametitle{Selection less effective in Neanderthals}
{\centering\includegraphics[width=0.9\linewidth]{Castellano-spectrum.png}\\}
\end{frame}

\begin{frame}
\begin{columns}
\column{0.5\textwidth}
\includegraphics[width=\linewidth]{Castellano-spectrum.png}
\column{0.5\textwidth}
\textcolor{blue}{Selection less effective in Neanderthals}

\bigskip

Many Neanderthal alleles have large effect on protein structure and
are probably deleterious.\\[1ex]
\mbox{}\hfill {\small (Castellano et al 2014)\\}
\end{columns}
\end{frame}

%\begin{frame}
%\begin{columns}
%\column{0.65\textwidth}
%\includegraphics[width=\linewidth]{Harris-hist.png}
%\column{0.35\textwidth}
%How many deleterious mutations would you expect?
%
%\bigskip
%
%How large would these deterious effects tend to be?
%
%\bigskip
%
%Simulation designed to answer these questions. (Harris and Nielsen 2015)
%\end{columns}
%\end{frame}

%\begin{frame}
%\frametitle{Simulated fitnesses of Neanderthals and moderns}
%\centering\includegraphics[height=0.8\textheight]{Harris-NeanFitA.png}\\
%\end{frame}

%\begin{frame}
%\frametitle{A few alleles of large effect, or many small effects?}
%\centering\includegraphics[height=0.8\textheight]{Harris-NeanFitC.png}\\
%\end{frame}

%\begin{frame}
%\frametitle{Most of the action is where $N_A s \approx 1$}
%
%Largest effect on fitness: $0.0001 < s < 0.001$
%
%\bigskip
%
%Or: $0.1 < N_A s < 1$, where $N_A$ is size of archaic population.
%
%\bigskip
%\pause
%
%Genetic disease dominated by alleles with small effect.
%\end{frame}

\begin{frame}
\frametitle{Summary}
\begin{itemize}
\item Archaic alleles are over-represented at immune loci in modern
  humans.
\item Modern humans needed help from archaics to fight Eurasian
  pathogens.
\item But many archaic alleles seem to have been deleterious.  
\item This may be because the small sizes of archaic populations
  allowed deleterious alleles to drift to high frequencies.
\end{itemize}
\end{frame}

\end{document}
