% -*-latex-*-
%\documentclass[pdftex,handout]{beamer} 
\documentclass[pdftex,12pt]{beamer} 
\usepackage{etex,pictex,graphicx,moreverb}
\newcommand{\G}{\mathcal{G}}
\newcommand{\Het}{\mathcal{H}}
\newcommand{\mutation}{$\cdot$} 
\newcommand{\OR}{\;\hbox{or}\;}
\newcommand{\AND}{\;\&\;}
\title{Alleles and Genotypes in Populations that Mate at Random}
\author{Alan R. Rogers}
\date{\today}
\setbeamercovered{transparent}
\begin{document}

\begin{frame}
I no longer use the slides that follow. They describe a statistical test,
which demonstrates a departure from HW, probably caused by ascertainment
bias. Too hard to follow, especially for students unfamiliar with
statistical tests.
\end{frame}

\begin{frame}
\frametitle{Python program that generated the data}
\listinginput{1}{../../src/hweq.py}
\end{frame}

\begin{frame}[containsverbatim]
\frametitle{Testing the Hardy-Weinberg hypothesis}

\begin{verbatim}
Genotypes for one HapMap SNP:
   2, 1, 2, 2, 2, 0, 2, 0, 0, 1, 0, 1, 2, 0, 1, 1, 2, 
   2, 2, 0, 2, 0, 1, 0, 0, 2, 0, 1, 0, 1, 0, 0, 1, 1, 
   2, 1, 0, 0, 2, 0, 0, 0, 0, 2, 0

p = 0.433333333333 H = 0.244444444444 E[H] = 0.49
Observed ChiSq = (H-E[H])^2/E[H] = 0.1239

Simulated values of ChiSq:
0.0012 0.0012 0.0000 0.0097 0.0044 0.0044 0.0097 0.0000 
0.0044 0.0085 0.0085 0.0036 0.0000 0.0153 0.0153 0.0000 
0.0153 0.0008 0.0000 0.0153 0.0169 0.0097 0.0036 0.0008 
0.0241 0.0085 0.0008 0.0012 0.0044 0.0000

ALL are lower than the observed value
\end{verbatim}
\end{frame}

\begin{frame}
\frametitle{Summary of 2008 HW tests w/ HapMap}
\begin{columns}
\column{0.5\textwidth}
\input{figpval}
\column{0.5\textwidth}
If hypothesis is true, all $p$-values are equally likely.

\pause
This graph should be flat!!

\pause
Conclusion: The Hardy-Weinberg hypothesis is false!!

\pause
Why?
\end{columns}
\end{frame}

\begin{frame}[containsverbatim]
\begin{verbatim}
Range                    Count
[     -inf ... -0.00001]     0
[ -0.00001 ...  0.19999]  1242 Many small values
[  0.19999 ...  0.40000]   383   
[  0.40000 ...  0.60000]   378
[  0.60000 ...  0.80001]     5 A few large ones
[  0.80001 ...  1.00001]     0
[  1.00001 ...      inf]     0
Total                     2008   
\end{verbatim}

What contributes to the variation?
\end{frame}

\begin{frame}
\frametitle{A statistical test}
\begin{itemize}
\item Calculate observed $p$ and $H$ from data.
\item Simulate 100 data sets by sampling from urn.
\item Calculate $H$ from each.
\item Upper tail: fraction of trials with simulated $H \geq \mbox{observed}$
\item Lower tail: fraction of trials with simulated $H \leq \mbox{observed}$
\item Reject at 0.05 level if either tail $\leq 0.025$.
\end{itemize}
\end{frame}

\begin{frame}[containsverbatim]
\frametitle{Example test results}
\begin{verbatim}
Data: ['AA', 'AA', 'AT', 'TA', 'TA', 'TT', 'AA', 'TA']
Observed heterozygosity = 0.5
Expected heterozygosity = 0.46875
Fraction of trials in upper tail (Hsim >= Hobs): 0.61
Fraction of trials in lower tail (Hsim <= Hobs): 0.65
\end{verbatim}
\end{frame}

\begin{frame}[containsverbatim]
\frametitle{Distribution of $p$; 1Mb of chromosome 7}
\begin{verbatim}
Range                    Count
[     -inf ... -0.00001]     0
[ -0.00001 ...  0.09999]    50    Distribution of p
[  0.09999 ...  0.19999]    67    should be uniform.
[  0.19999 ...  0.30000]    66
[  0.30000 ...  0.40000]   114
[  0.40000 ...  0.50000]   223    1% of loci rejected
[  0.50000 ...  0.60000]   214    at 5% level.
[  0.60000 ...  0.70000]   180
[  0.70000 ...  0.80001]    95
[  0.80001 ...  0.90001]   107
[  0.90001 ...  1.00001]    40
[  1.00001 ...      inf]     0
Total                     1156
\end{verbatim}
\end{frame}


\begin{frame}
\frametitle{Too good to be true}
\begin{itemize}
\item Distribution of $p$-values should be flat.
\item Too many are btw 0.2 and 0.8.
\item At 5\% significance, we should reject 5\%, not 1\%.
\item What is going on?
\end{itemize}
\end{frame}

\end{document}
