% -*-latex-*-
%\documentclass[handout]{beamer} 
\documentclass{beamer} 
\usepackage{etex}

\let\latexput\put
\usepackage{pictex}
\let\pictexput\put
\let\put\latexput

\newcommand{\G}{\mathcal{G}}
\newcommand{\Het}{\mathcal{H}}
\newcommand{\mutation}{$\cdot$} 
\newcommand{\OR}{\;\hbox{or}\;}
\newcommand{\AND}{\;\&\;}
\newdimen\thusfar
\newdimen\plotht
\newdimen\plotwd
\newdimen\plotsp
\newdimen\xunit
\newdimen\yunit
\newdimen\offsety
\title{The History of Population Size from Whole Genomes}
\author{Alan R. Rogers}
\date{\today}
\begin{document}

\frame{\titlepage}

\begin{frame}
\frametitle{Simulated gene genealogy of a sample of size 50 from a
  population of constant size}
\centering\input{figeqtree}\\

\bigskip

\begin{itemize}
\item Short terminal branches; long basal ones.
\item Large samples tell us about recent past.
\item Not necessary for ancient past.
\end{itemize}
\end{frame}

\begin{frame}
\begin{columns}
\column{0.5\textwidth}
\input{figgro}
\column{0.5\textwidth}
{\color{blue}Effect of a population explosion}\\

\bigskip

Middle: genealogy of 50 individuals; dots are mutations.\\

\bigskip

1 mutational diff per time unit

\bigskip

Bottom: $\circ$ = simulated data, line = theory.\\

\bigskip

Wave peaks at population expansion.\\
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Skyline Plot}
\begin{columns}
\column{0.5\textwidth}
\includegraphics[width=\linewidth]{skyline.png}\\
\mbox{}\hfill\footnotesize (Drummond et al 2005)\\
\column{0.5\textwidth}
\begin{itemize}
\item Use mutations to estimate length of each interval.
\item Long intervals imply large population size.
\item Won't work with nuclear DNA: too few mutations per tree
\end{itemize}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Nuclear genome}

\begin{itemize}
\item Huge amounts of data.
\item Recombination makes previous methods unusable.
\end{itemize}
\end{frame}

\begin{frame}
\begin{columns}
\column{0.6\linewidth}
{\centering\input{figgrospec}\\}
\column{0.4\linewidth}
\textcolor{blue}{\Large Site frequency spectrum}

\bigskip

The spectrum is useful with nuclear as well as mitochondrial DNA.

\bigskip

Population growth inflates the number of singletons. Bars show
simulated spectrum; filled circles show expectation under constant
population size.

\bigskip

To estimate history, we need a theory that works under complex
population histories.

\end{columns}
\end{frame}

\begin{frame}
  \frametitle{We do have such a theory}
  \begin{columns}
    \column{0.6\textwidth}
    \includegraphics[width=\linewidth]{Wooding-fig1.pdf}
    \column{0.4\textwidth}
    \raggedleft
    Row~1: 3 histories of population size

    \bigskip

    Row~2: Prob that there are $k$ lineages at time $t$ in a sample of
    size 5.

    \bigskip

    Row~3: Expected lengths each coalescent intervals.

    \bigskip

    Row~4: Expected spectrum

    \bigskip

    {\footnotesize Griffiths \& Tavar{\'e} (1998); Wooding \& Rogers (2002)\\}
  \end{columns}
\end{frame}  

\begin{frame}
\frametitle{$\partial$a$\partial$i: inferences from the site frequency spectrum}
\includegraphics[width=\linewidth]{gutenkunst.png}\\
\mbox{}\hfill\footnotesize(Gutenkunst et al 2009)\\
\end{frame}

\begin{frame}
\frametitle{Stairway plot (Liu \& Fu 2015)}

\begin{itemize}
\item uses site frequency spectrum
\item no need for phased data
\item can deal with samples of hundreds of individuals
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Stairway Plot results}
{\centering\includegraphics[height=0.8\textheight]{stairway.png}\\
\footnotesize 1000-Genomes data; (Liu \& Fu 2015)\\}
\end{frame}

\begin{frame}
\frametitle{Also: recombination is our friend}
\begin{columns}
\column{0.5\linewidth}
\includegraphics[width=\linewidth]{crossover.pdf}
\column{0.5\linewidth}
Useful data began to appear in about 2000.

\bigskip

Crossovers shuffle DNA

\bigskip

Each chromosome has many gene genealogies, which vary in length.
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Two (hypothetical) loci in a single diploid genome}
{\centering\input{figtwotrees}\\}

\bigskip

\begin{itemize}
\item MRCA: most recent common ancestor
\item Gene trees vary in length across the genome.
\item Mutation ({\tiny$\bullet$}) is more likely on a deep gene tree.
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{MRCA varies along the chromosome}
{\centering\input{figmrcavary}\\}
\begin{itemize}
%\item MRCA is depth of gene tree.
\item Circles: nucleotide sites that differ (are \emph{heterozygous})
  in a single diploid sample.
\item Heterozygous sites are denser where gene tree is deep.
\item Population size $\rightarrow$ length of
  MRCA segments and genetic variation within segments.
\item PSMC uses this pattern to estimate population history
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{PSMC is accurate from 20~ky to 3~my ago.}
\centering
\includegraphics[width=\linewidth]{LiDurbin-sim.png}\\

\mbox{}\hfill{\small(Li and Durbin 2011)\\}
\end{frame}

\begin{frame}
\frametitle{PSMC estimates from autosomes}
\begin{columns}
\column{0.65\linewidth}
\includegraphics[width=\linewidth]{LiDurbin-esta.png}
\column{0.4\linewidth} $\uparrow$ 2~mya (origin of \emph{Homo});
$\uparrow$ 200~kya (origin of modern humans); $\uparrow$ 20~kya
(beginning of Holocene).
\end{columns}

\bigskip

Eurasian/African split 150~kya. 

\bigskip

African bottleneck short and shallow.
\end{frame}

\begin{frame}
\frametitle{PSMC estimates from X chromosomes}
\includegraphics[width=\linewidth]{LiDurbin-estb.png}
\end{frame}

\begin{frame}
\frametitle{PSMC with Neanderthal as well as Denisova}
\includegraphics[width=\linewidth]{Prufer-pophist.jpg}\\
\mbox{}\hfill\footnotesize(Pr{\"u}fer et al 2014)\\
\end{frame}

\begin{frame}
  \frametitle{All 3 high-coverage archaic genomes}
{\centering\includegraphics[width=0.9\linewidth]{Chagyrskaya-psmc.png}\\}

\bigskip

Horizontal axis shows time backwards from time of fossil's
death. Curves of younger fossil (Vindija) is shifted to right.\\
\mbox{}\hfill\footnotesize(Mafessoni et al 2022)\\
\end{frame}

\begin{frame}
  \frametitle{What if the population is subdivided? (part 1)}
  Suppose there are $K$ local groups, each of size $N$, which exchange
  migrants. We sample a single genome from some group. In the recent
  past, the hazard of a coalescent event is $h(t) = 1/2N$, and effective
  population size ($1/h(t)$) is $2N$.

  \bigskip

  Farther back in time, the ancestors of the 2 gene copies in our
  diploid sample may be in different groupss. At time $t$,
  \[
  h(t) = \begin{cases}
    1/2N & \hbox{if they're in the same group}\\
    0  & \hbox{if they're in different groups}
    \end{cases}
  \]
\end{frame}

\begin{frame}
  \frametitle{What if the population is subdivided? (part 2)}

  In the distant past, those loci that have still not coalesced tend
  to be those at which the two lineages haven't spent much time
  together in the same local group.

  \bigskip

  Consequently, the two lineages are \emph{less likely} to be in the
  same group than are two drawn at random from the population as a
  whole.

  \bigskip

  This means that in the distant past, $N_e$ is greater than the
  actual population size.
\end{frame}

\begin{frame}
\frametitle{PSMC gives misleading signal of decline in subdivided
  populations, even if population size is constant (Mazet et al 2014).}
{\centering\includegraphics[width=0.8\textwidth]{island}\\}
$M$ is twice the number of immigrants per group per generation;
horizontal black line is true population size.
\end{frame}

\begin{frame}
\frametitle{Population structure cannot explain the archaic decline} 
\includegraphics[width=\linewidth]{Chagyrskaya-psmc.png}

Decline due to population structure must be either faster or larger
than that seen in archaic data.

\bigskip

There must have been a change either in the number of individuals or
in the rate or pattern of mobility among groups.
\end{frame}

\begin{frame}
\frametitle{Once again: simulated gene genealogy of a sample of
  size 50 from a population of constant size}
\centering\input{figeqtree}\\

\bigskip

To estimate the \emph{recent} history of population size, you need
larger samples.
\end{frame}

\begin{frame}
\frametitle{MSMC: using multiple genomes}
\centering\includegraphics[width=0.9\linewidth]{SchiffelsDurbin-esta.png}\\
\end{frame}

\begin{frame}
\frametitle{Very recent population growth is tough}
\input{fig100g}
\end{frame}

\begin{frame}
\frametitle{SMC++: combines PSMC and spectrum (Terhorst, Kamm, \& Song 2017)}
\includegraphics[width=\textwidth]{Terhorst-pophist.jpg}
\end{frame}

\begin{frame}
\frametitle{Separation times (Terhorst, Kamm, \& Song 2017)}
\includegraphics[width=\textwidth]{Terhorst-septime.jpg}
\end{frame}

\begin{frame}
\frametitle{Methods for studying the history of population size}
\begin{itemize}
  \item Large sample are useful for studying recent history; small
    samples are sufficient for ancient history.
  \item The site frequency spectrum can be used with large samples and
    is therefore useful in studying recent history. However, it
    ignores the information provided by LD.
  \item PSMC uses a single diploid genome. It is good for ancient
    history but not for recent history.
  \item MSMC uses several diploid genomes and has more power at recent
    time scales.
  \item SMC++ combines two sources of information: LD and the site
    frequency spectrum. It has power across a wide range of time
    scales. 
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Conclusions about human history}
\begin{itemize}
\item History of population size affects depth of gene trees, genetic
  variation, and length of MRCA segments.
\item We can use these facts to infer the history of population size.
\item Human population has varied in size over past 3~my.
\item Bottleneck during last ice age, ending 20~kya.
\item African bottleneck was shorter and shallower.
\item Eurasian/African split 150~kya.
\item European/Asian split 20~kya.
\item Effect of geographic population structure looks like population decline.
\end{itemize}
\end{frame}

\end{document}

\begin{frame}
\frametitle{Alu insertions}
\begin{itemize}
\item Insert into random spots in genome.
\item These are rare events: 0.05 per genome per generation.
\item Most likely to be found in deep gene trees.
\item A sample of Alu insertions is a sample of unusually deep gene
  trees.
\item Gene diversity should be high in neighborhood of insertion.
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Gene diversity near Alus vs at random locations}

\centering
  \includegraphics[height=0.8\textheight]{deeprand.pdf}\\ \scriptsize
Distributions are significantly different ($p\ll 10^{-30}$,
21X2 $\mathcal{X}^2$, Huff et al 2010).\\
\end{frame}


\begin{frame}
\frametitle{Genealogy of two genes}
{\centering
\input{figpaircoal}\\[10pt]}
$E[t_1] = 2N$~generations, where $N$ is effective population size.
\end{frame}

\begin{frame}
\frametitle{Hazards and lengths for a single pair of lineages}
\begin{columns}
\column[T]{0.48\linewidth}
Coalescent event: $h_c = 1/2N$
\column[T]{0.02\linewidth}
\pause
\rule{0.5pt}{0.6cm}
\column[T]{0.48\linewidth}
Mutation: $h_m = 2u$
\end{columns}
\pause
\noindent\rule{\linewidth}{0.5pt}

Hazard of an event of either type
\[
h = h_c + h_m = 1/2N + 2u
\]
\pause
\noindent\rule{\linewidth}{0.5pt}

Expected time until an event of either type
\begin{eqnarray*}
1/h &=& 1/(1/2N + 2u)\\
&=& \frac{2N}{1 + 4Nu}\\
&\approx& 2N \qquad\qquad\qquad \hbox{if $4Nu \ll 1$.}
\end{eqnarray*}
\end{frame}

\begin{frame}
\frametitle{Conditional on a single insertion}
\centering
\input{figmutcoal}\\

\bigskip

$E[t_1+t_2] \approx 4N$~generations.

\bigskip

Conditioning on insertion doubles length of genealogy (Tajima 1983).
\end{frame}

\begin{frame}
\frametitle{Nucleotide diversity decays with distance from Alu}
\centering
\includegraphics[width=\textwidth]{Huff-scatter.pdf}\\
{\mbox{}\hfill\footnotesize (Huff et al 2010)\\}
\pause
\bigskip
Can we use this to study history?
\end{frame}

\begin{frame}
\frametitle{Some hypotheses about population history}
\begin{center}
\includegraphics[height=0.85\textheight]{allhist-crop.pdf}\\
\end{center}
\end{frame}

\begin{frame}
\frametitle{Closeup of bottleneck}
\begin{columns}
\column{0.5\textwidth}
\includegraphics[width=\textwidth]{btlhist-crop.pdf}\\
\column{0.5\textwidth}
\begin{itemize}
\item Bottleneck is brief but severe.
\item Removes much of genetic variation.
\item Generates wave in mismatch distribution.
\end{itemize}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Data}
\begin{itemize}
\item Two complete genomes
\item 639 Alu insertions
\item DNA sequence variation w/i 2.5~kb of insertion, on either side
\item nucleotide diversity: fraction of sites that differ between two genomes.
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Statistical methods}
\begin{itemize}
\item Each hypothesis involves three parameters:
\begin{enumerate}
  \item Size $N_A$ of ancient population.
  \item Size $N_M$ of later population.
  \item Time $t$ since change in size.
\end{enumerate}
\item Simulate to approximate sampling distribution of gene diversity
  under each hypothesis.
\item Consider hypotheses covering a grid of $(N_A, N_M, t)$ values.
\item Likelihood ratio tests define confidence region.
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Quantile-quantile plots for random genomic regions}
{\centering
\includegraphics[height=0.85\textheight]{qqran-crop.pdf}\\}
\end{frame}

\begin{frame}
\frametitle{Quantile-quantile plots for regions near Alus}
{\centering
\includegraphics[height=0.85\textheight]{qqdeep-crop.pdf}\\}
\end{frame}

\begin{frame}
\frametitle{Point estimates and marginal confidence intervals}
\[
\begin{array}{rclc}
\hat t &=& 2.4      & (1.8{-}3)\\
\hat N_M &=& 17,000  & (16,200{-}17,500)\\
\hat N_A &=& 33,000 & (29,000{-}52,000)\\
\end{array}
\]
Using revised mutation rate, which is 1/2 as large as the one used by
Huff, 2010.

\bigskip\footnotesize

Ref: Huff, CD et al. 2010.  Mobile Elements Reveal Small Population
Size in the Ancient Ancestors of \emph{Homo sapiens}. PNAS 107(5):
2147--2152.
\end{frame}

\begin{frame}
\frametitle{Current research (Xing et al)}

Studied 4 genomes (2 European and 2 African) from the 1000-genomes
project.

\begin{center}
\begin{tabular}{lccc}
   & $t$ & $N_M$ & $N_A$\\
\hline
European  & 2.5 (2--2.8)   &  14,000 & 61,600 (43,400--70,000)\\
African   & 2.9 (2.5--3.3) &  15,500 & 57,400 (46,500--74,500)\\
\end{tabular}
\end{center}

\bigskip\footnotesize

Ref: Xing, Jinchuan, et al. In prep. Mobile elements demonstrate that
\emph{Australopithecus} effective size was twice that of \emph{Homo}.
(Used with permission.)
\end{frame}

\begin{frame}
\frametitle{SMC++: combines PSMC and spectrum (Terhorst, Kamm, \& Song 2017)}
\includegraphics[width=\textwidth]{Terhorst-pophist.jpg}
\end{frame}

\begin{frame}
\frametitle{Separation times (Terhorst, Kamm, \& Song 2017)}
\centering\includegraphics[height=0.8\textheight]{Terhorst-septime.jpg}\\
\end{frame}

\begin{frame}
\frametitle{Summary}
\begin{itemize}
\item Rare events provide a window into the distant past.
\item The human population declined 2--3~million years ago, at the
  origin of the genus \emph{Homo}.
\end{itemize}
\end{frame}

\end{document}
