%-*-latex-*-
\begin{figure}
\begin{center}
\mbox{%
\beginpicture
\headingtoplotskip=0mm
\valuestolabelleading=0.2\baselineskip
%%%%%%%%%%%%%%%% row 1 col 1 %%%%%%%%%%%%%%%%
%point at 2.125in 0in
\setcoordinatesystem units <2.5in, 1.51in> point at 0.85 0
\setplotarea x from 0 to 0.5, y from 0 to 0.83
\axis left label {\lines{Frac.\cr of\cr sites}}  /
\axis bottom ticks numbered from 0.0 to 0.5 by 0.5 /
\plotheading{MtDNA$^{a,b}$}
\put{$\begin{array}{rcl}
      K &=& 636\\ % number of sequences in sample
      s &=& 226\\  % number of segregating sites
       \end{array}$} [tr] at 0.5 0.83
%Expected site frequency spectrum
\multiput {\footnotesize$\bullet$} at
0.013 0.475 0.038 0.104 0.062 0.062 0.088 0.045 0.113 0.036
0.138 0.030 0.163 0.026 0.188 0.023 0.213 0.020 0.237 0.019
0.263 0.016 0.288 0.016 0.312 0.015 0.338 0.014 0.363 0.013
0.388 0.013 0.413 0.012 0.438 0.012 0.463 0.011 0.488 0.011
/
%Observed site frequency spectrum
\sethistograms
\plot
0.000 0.000 0.025 0.827 0.050 0.075 0.075 0.044 0.100 0.013
0.125 0.013 0.150 0.004 0.175 0.004 0.200 0.004 0.225 0.000
0.250 0.000 0.275 0.000 0.300 0.004 0.325 0.000 0.350 0.004
0.375 0.000 0.400 0.004 0.425 0.000 0.450 0.000 0.475 0.000
0.500 0.000
/
%%%%%%%%%%%%%%%% row 1 col 2 %%%%%%%%%%%%%%%%
% point at 0.625in 0in
\setcoordinatesystem units <1.25in, 1.47in> point at 0.5 0
\setplotarea x from 0 to 1, y from 0 to 0.85
\axis bottom ticks numbered from 0 to 1 by 1 /
\plotheading{Y$^{b,c}$}
\put{$\begin{array}{rcl}
      K &=& 718\\ % number of sequences in sample
      s &=& 20\\  % number of segregating sites
       \end{array}$} [tr] at 1 0.85
%Expected site frequency spectrum
\multiput {\footnotesize$\bullet$} at
0.033 0.349
0.100 0.161
0.167 0.100
0.233 0.071
0.300 0.055
0.367 0.045
0.433 0.038
0.500 0.032
0.567 0.029
0.633 0.026
0.700 0.024
0.767 0.022
0.833 0.020
0.900 0.017
0.967 0.010                      
/
%Observed site frequency spectrum
\sethistograms
\plot
0.000 0.000
0.067 0.850
0.133 0.050
0.200 0.050
0.267 0.000
0.333 0.000
0.400 0.000
0.467 0.000
0.533 0.000
0.600 0.000
0.667 0.000
0.733 0.000
0.800 0.000
0.867 0.000
0.933 0.000
1.000 0.000
/                                              
%%%%%%%%%% row 1 col 3 %%%%%%%%%%%%%%%%
%point at -0.875in 0in
\setcoordinatesystem units <2.5in, 1.52in> point at -0.35 0
\setplotarea x from 0 to 0.5, y from 0 to 0.82
\axis bottom ticks numbered from 0.0 to 0.5 by 0.5 /
\plotheading{Xq13.3$^d$}
\put{$\begin{array}{rcl}
      K &=& 69\\ % number of sequences in sample
      s &=& 33\\  % number of segregating sites
       \end{array}$} [tr] at 0.5 0.82
%Expected site frequency spectrum
\multiput {\footnotesize$\bullet$} at
0.062 0.590
0.188 0.178
0.312 0.103
0.438 0.090         
/
%Observed site frequency spectrum
\sethistograms
\plot
0.000 0.000
0.125 0.818
0.250 0.121
0.375 0.061
0.500 0.000
/
%%%%%%%%%%%%%%%% row 2 col 1 %%%%%%%%%%%%%%%%
% point at 2.125in 1.85in
\setcoordinatesystem units <2.5in, 2.272in> point at 0.85 0.814
\setplotarea x from 0 to 0.5, y from 0 to 0.55
\axis left label {\lines{Frac.\cr of\cr sites}} /
\axis bottom ticks numbered from 0.0 to 0.5 by 0.5 /
\plotheading{$\beta$-globin$^e$}
\put{$\begin{array}{rcl}
      K &=& 253\\ % number of sequences in sample
      s &=& 33\\  % number of segregating sites
       \end{array}$} [tr] at 0.5 0.55
%Expected site frequency spectrum
\multiput {\footnotesize$\bullet$} at
0.025 0.516
0.075 0.125
0.125 0.071
0.175 0.057
0.225 0.046
0.275 0.036
0.325 0.034
0.375 0.031
0.425 0.026
0.475 0.026           
/
%Observed site frequency spectrum
\sethistograms
\plot
0.000 0.000
0.050 0.545
0.100 0.061
0.150 0.000
0.200 0.030
0.250 0.000
0.300 0.030
0.350 0.182
0.400 0.061
0.450 0.030
0.500 0.061
/                 
%%%%%%%%%%%%%%%% row 2 col 2 %%%%%%%%%%%%%%%%
% point at 0.625in 1.85in
\setcoordinatesystem units <2.5in, 3.125in> point at 0.25 0.592
\setplotarea x from 0 to 0.5, y from 0 to 0.4
\axis bottom ticks numbered from 0.0 to 0.5 by  0.5 /
\plotheading{Misc X$^f$}
\put{$\begin{array}{rcl}
      K &=& 10\\ % number of sequences in sample
      s &=& 20\\  % number of segregating sites
       \end{array}$} [tr] at 0.5 0.4
%Expected site frequency spectrum
\multiput {\footnotesize$\bullet$} at
0.050 0.393
0.150 0.216
0.250 0.157
0.350 0.128
0.450 0.071
/
%Observed site frequency spectrum
\sethistograms
\plot
0.000 0.000
0.100 0.400
0.200 0.200
0.300 0.100
0.400 0.250
0.500 0.050
/                                       
%%%%%%%%%%%%%%%% row 2 col 3 %%%%%%%%%%%%%%%%
% point at -0.875in 1.85in
\setcoordinatesystem units <2.5in, 1.89in> point at -0.35 0.979
\setplotarea x from 0 to 0.5, y from 0 to 0.66
\axis bottom ticks numbered from 0.0 to 0.5 by 0.5 /
\plotheading{SNP$^g$}
\put{$\begin{array}{rcl}
      K &=& 147\\ % number of sequences in sample
      s &=& 870\\ % number of segregating sites
       \end{array}$} [tr] at 0.5 0.66
%Expected site frequency spectrum
\multiput {\footnotesize$\bullet$} at
0.050 0.613
0.150 0.143
0.250 0.095
0.350 0.079
0.450 0.070
/
%Observed site frequency spectrum
\sethistograms
\plot
0.000 0.000
0.1 0.654
0.2 0.136
0.3 0.083
0.4 0.070
0.5 0.057
/                 
%%%%%%%%%%%%%%%% row 3 col 1 %%%%%%%%%%%%%%%%
%point at 2.125in 3.7in
\setcoordinatesystem units <2.5in, 2.5in> point at 0.85 1.48
\setplotarea x from 0 to 0.5, y from 0 to 0.5
\axis left invisible label {\lines{Frac.\cr of\cr sites}} /
\axis bottom 
  label {$p$} 
  ticks numbered from 0.0 to 0.5 by 0.5 /
\put{$\begin{array}{rcl}
      K &=& 142\\ % number of sequences in sample
      s &=& 88\\  % number of segregating sites
       \end{array}$} [tr] at 0.5 0.5
\plotheading{LPL$^h$}
%Expected site frequency spectrum
\multiput {\footnotesize$\bullet$} at
0.025 0.478
0.075 0.128
0.125 0.082
0.175 0.061
0.225 0.058
0.275 0.044
0.325 0.040
0.375 0.037
0.425 0.036
0.475 0.035
/
%Observed site frequency spectrum
\sethistograms
\plot
0 0
0.050 0.386
0.100 0.125
0.150 0.057
0.200 0.102
0.250 0.068
0.300 0.057
0.350 0.080
0.400 0.068
0.450 0.011
0.500 0.045
/
%%%%%%%%%%%%%%%% row 3 col 2 %%%%%%%%%%%%%%%%
%point at 1.5in 3.7in
\setcoordinatesystem units <1.25in, 2.08in> point at .5 1.78
\setplotarea x from 0 to 1, y from 0 to 0.6
\axis bottom label {$p$} ticks numbered from 0 to 1 by  1 /
\put{$\begin{array}{rcl}
      K &=& 250/860\\ % number of sequences in sample
      s &=& 33\\  % number of segregating sites
       \end{array}$} [tr] at 1 0.6
\plotheading{Dystrophin$^i$}
%Expected site frequency spectrum
\multiput {\footnotesize$\bullet$} at
0.025 0.593 0.075 0.094 0.125 0.055 0.175 0.039 0.225 0.030
0.275 0.025 0.325 0.021 0.375 0.018 0.425 0.016 0.475 0.014
0.525 0.013 0.575 0.012 0.625 0.011 0.675 0.010 0.725 0.009
0.775 0.009 0.825 0.008 0.875 0.008 0.925 0.007 0.975 0.007
/
%Observed site frequency spectrum
\sethistograms
\plot
0.000 0.000
0.050 0.389 
0.100 0.111 
0.150 0.056 
0.200 0.000 
0.250 0.028
0.300 0.000 
0.350 0.083 
0.400 0.056 
0.450 0.028 
0.500 0.028
0.550 0.000 
0.600 0.028 
0.650 0.083 
0.700 0.028 
0.750 0.000
0.800 0.000 
0.850 0.000 
0.900 0.028 
0.950 0.056 
1.000 0.000
/
%%%%%%%%%%%%%%%% row 3 col 3 %%%%%%%%%%%%%%%%
%point at -0.875in 3.7in
\setcoordinatesystem units <2.5in, 2.273in> point at -0.35 1.63
\setplotarea x from 0 to 0.5, y from 0 to 0.55
\axis bottom 
  label {$p$} ticks
  numbered from 0.0 to 0.5 by 0.5 /
\plotheading{PDHA1$^j$}
\put{$\begin{array}{rcl}
      K &=& 35\\ % number of sequences in sample
      s &=& 24\\  % number of segregating sites
       \end{array}$} [tl] at 0 0.55
%Expected site frequency spectrum
\multiput {\footnotesize$\bullet$} at
0.028 0.250
0.083 0.217
0.139 0.124
0.194 0.089
0.250 0.072
0.306 0.061
0.361 0.053
0.417 0.048
0.472 0.044
/
%Observed site frequency spectrum
\sethistograms
\plot
0.000 0.000
0.056 0.125
0.111 0.250
0.167 0.042
0.222 0.042
0.278 0.000
0.333 0.500
0.389 0.000
0.444 0.000
0.500 0.042
/
\endpicture%
}
\end{center}
\caption{Site frequency spectra.  The open rectangles in each panel
  show observed spectra; the bold dots show the spectra expected under
  the infinite sites model with no selection and constant population
  size.  $K$ is the number of chromosomes sampled, $s$ is the number
  of segregating sites, and $p$ is the frequency of the mutant allele
  (where ancestral state could be determined) or the rarest allele
  (where ancestral state is unknown).
  Sources:
$^a$\protect\citep{Jorde:AJH-57-523},
$^b$\protect\citep{Harpending:PNA-95-1961},
$^c$\protect\citep{Underhill:GR-7-996},
$^d$\protect\citep{Kaessmann:NG-22-78}, 
$^e$\protect\citep{Harding:AJH-60-722},
$^f$\protect\citep{Nachman:G-150-1133},
$^g$\protect\citep{Halushka:NG-22-239},
$^h$\protect\citep{Clark:AJH-63-595},
$^i$\protect\citep{Zietkiewicz:JME-47-146},
$^j$\protect\citep{Harris:PNA-96-3320}
}
\label{fig.spec}
\end{figure}

%%% Local Variables: 
%%% mode: latex
%%% TeX-master: t
%%% End: 
