% -*-latex-*-
%\documentclass[dvips,handout]{beamer}  %% handout
\documentclass[dvips,12pt]{beamer}    %% presentation
\usepackage{etex,pictex}
%\usepackage{pstricks,pst-plot}
%\setlength{\topmargin}{-1.7in}
\setbeamercovered{transparent}
\title{Diffusion Wave Hypothesis}
\author{Alan R. Rogers}
\date{\today}
\begin{document}
\frame{\titlepage}

\frame{\frametitle{Neandertal \& Modern}
\begin{center}
 \includegraphics[height=0.7\textheight]{nean_mod.eps}
\end{center}
}

\begin{frame}
\frametitle{Moderns invade Eurasia}
\centering
 \includegraphics[width=\textwidth]{modhummap.eps}
\end{frame}

\frame{\frametitle{Moderns made fancy points}
\centering
 \includegraphics[width=\textwidth]{solutrean.eps}
}

\frame{\frametitle{Spear throwers}
\centering
 \includegraphics[height=0.7\textheight]{atlatl.ps}
}

\frame{\frametitle{Art}
\centering
 \includegraphics[height=0.7\textheight]{lascaux2.eps}
}

%\frame{\frametitle{Pace of change}
%\begin{itemize}
%\item slow before moderns arrive
%\item fast thereafter
%\end{itemize}
%}

\frame{\frametitle{How did modern humans evolve?}
In the 1980s and 90s, there were two main hypotheses
\begin{itemize}
\item Multiregional
\item Replacement
\end{itemize}
}

\begin{frame}
\frametitle{Multiregional hypothesis}
\begin{columns}
\column{0.5\textwidth}
\includegraphics[height=0.7\textheight]{multireg.ps}
\column{0.5\textwidth}
\begin{itemize}
\item<+-| alert@+> \emph{Homo erectus} expands into Eurasia 1.8~mya
\item<+-| alert@+> Strong gene flow
\item<+-| alert@+> Favorable mutations spread in every direction.
\item<+-| alert@+> Moderns have no geographic origin.
\end{itemize}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Replacement hypothesis}
\begin{columns}
\column{0.5\textwidth}
\includegraphics[height=0.7\textheight]{replacement.ps}
\column{0.5\textwidth}
\begin{itemize}
\item<+-| alert@+> \emph{Homo erectus} expands into Eurasia 1.8~mya
\item<+-| alert@+> Moderns evolve in Africa by 100~kya
\item<+-| alert@+> Expand into Eurasia btw 100 and 30~kya
\item<+-| alert@+> No mixing with archaics
\end{itemize}
\end{columns}
\end{frame}

\frame{\frametitle{Mitochondrial story, circa 1996}
This problem has been studied using bones, stones, and genes.
Here's how I would have summarized the genetic evidence:
\begin{itemize}
\item<+-| alert@+> Population expansion 50--100~kya
\item<+-| alert@+> Early population $\leq 5000$ females
\item<+-| alert@+> Late population $\geq \hbox{1,000,000}$ females
\item<+-| alert@+> Consistent with replacement hypothesis
\item<+-| alert@+> but not multiregional hypothesis
\end{itemize}
}

\begin{frame}
\frametitle{Nuclear genes}
Since 1997, we have relied more and more on nuclear genes.
\end{frame}

\begin{frame}
\frametitle{Some nuclear loci have shallow genealogies}
\begin{columns}
\column{0.65\textwidth}
\includegraphics[width=\textwidth]{underhilltree.eps}\\
\textsc{\small (Underhill \& al 2000)}
\column{0.35\textwidth}
\begin{block}{Y chromosome}
49~kyr old

\pause
Old branches are short;
\pause
suggests expansion.

\pause
Supports replacement hypothesis. 
\end{block}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Others have deep genealogies}
\begin{columns}
\column{0.65\textwidth}
\includegraphics[width=\textwidth]{pdha1tree.eps}
\column{0.35\textwidth}
\begin{block}{PDHA1 locus {\small (Harris \& Hey 1999)}}
\pause
Nearly 2~myr old.

\pause 
Old branches long.

\pause
Suggests subdivision.

\pause
Supports multiregional hypothesis 
\end{block}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{The puzzle}
Some nuclear loci support one hypothesis; 
\pause
some support the other.

\bigskip
\pause
The truth must somehow encompass both.
\end{frame}

\begin{frame}
\frametitle{Diffusion wave authors}
Vinayak Eswaran\hfill Henry Harpending\\
\begin{columns}
\column{0.5\textwidth}
\includegraphics[width=\textwidth]{VEswaran.eps}%
\column{0.5\textwidth}
\includegraphics[width=\textwidth]{hch.eps}%
\end{columns}
\end{frame}

%\begin{frame}
%\frametitle{Maybe a trickle of admixture?}
%\centering
%\includegraphics[width=\textwidth]{models-hodgson08.eps}
%\end{frame}

\begin{frame}
\frametitle{Diffusion wave hypothesis}
\begin{itemize}
\item \emph{Homo erectus} expands into Eurasia 1.8~mya
\pause
\item Moderns evolve in Africa by 100~kya
\pause
\item Expand into Eurasia btw 100 and 30~kya
\pause
\item Some mixing with archaics
\pause
\item Co-adapted gene complex reduces fitness of hybrids.
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Co-adapted gene complexes}
\begin{itemize}
\item Hybrids often have low fitness
\pause
\item Especially when population is deeply subdivided
\pause
\item Probably because each sub-population has evolved a
     different ``co-adapted gene complex.''
\item Co-adapted gene complex: a group of alleles at
     different loci that work well together, but not apart.
\item For example, the parts in your Toyota work well
     together, but don't install them in your Ford.
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Copepod \emph{Tigriopus californicus}}
\begin{columns}
\column{0.5\textwidth}
\includegraphics[width=\textwidth]{tigriopus.ps}
\column{0.5\textwidth}
\begin{itemize}
\pause
\item Live in tide pools
\pause
\item Isolated populations
\pause
\item Hybrids have low fitness
\pause
\item Different complexes of coadapted alleles
\end{itemize}
\end{columns}
\end{frame}


\begin{frame}
\frametitle{Modelling a co-adapted gene complex}
\begin{itemize}
\item Several unlinked loci
\pause
\item Each locus has a ``modern'' and an ``archaic'' allele.
\item Modern individuals have 2 copies of the modern
     allele at each locus.  All others are archaic.
\pause
\item Moderns have a selective advantage.
\pause
\item Advantage disappears if you have even 1 copy of an
     archaic allele
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Neutral loci}
\begin{itemize}
\item
There are 40 unlinked neutral loci, in addition to the co-adapted gene
complex. 
\pause
\item
The statistics we report are based on these neutral loci, not on the
co-adapted gene complex.
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Progress of diffusion wave}
\input{figdiffwave}
\end{frame}

\begin{frame}
\frametitle{Fraction of neutral loci that are African, after diffusion wave}
\input{figdw}
\end{frame}

\begin{frame}
\frametitle{Diffusion wave predicts archaic admixture at some
(but not all) loci.} 

\begin{itemize}
\pause
\item At few loci if adaptation is complex.
\pause
\item At many loci if adaptation is simple.
\end{itemize}
\pause
Just the sort of confusion that we see in real data.

\pause
\medskip
Next$\ldots$ the evidence
\end{frame}

\begin{frame}
\frametitle{Tajima's D}
\begin{itemize}
\item  a statistic that is sensitive
  both to selection and population growth.

\pause
\item behaves differently under the replacement,
  multiregional, and diffusion wave hypotheses.
\end{itemize}
\end{frame}

\begin{frame}
\frametitle{Replacement hypothesis}
 \begin{columns}
\column{0.7\textwidth}
 \includegraphics[width=\textwidth]{piDrepl.ps}
\column{0.3\textwidth}
\raggedright
Simulated Tajima's D 
\begin{itemize}
\pause
\item Many near --2.\\
\pause
\item None much above --1.
\end{itemize}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Multiregional hypothesis}
\begin{columns}
\column{0.7\textwidth}
 \includegraphics[width=\textwidth]{piDmre.eps}
\column{0.3\textwidth}
\raggedright
Simulated Tajima's D 
\begin{itemize}
\pause
\item Symmetric about zero\\
\pause
\item Would be $>0$ with less migration
\end{itemize}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Multiregional hypothesis again}
\begin{columns}
\column{0.7\textwidth}
 \includegraphics[width=\textwidth]{piDmre2.ps}
\column{0.3\textwidth}
\raggedright
Simulated Tajima's D 
\begin{itemize}
\item With more population structure\\
\pause
\item Average above zero.\\
\end{itemize}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Diffusion Wave Hypothesis}
\begin{columns}
\column{0.7\textwidth}
\includegraphics[width=\textwidth]{piDdiffwave.ps}
\column{0.3\textwidth}
\raggedright
Simulated Tajima's D 
\begin{itemize}
\pause
\item   From --2 to +1.5.
\pause
\item   Most are negative.
\end{itemize}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Reality}
\begin{columns}
\column{0.7\textwidth}
 \includegraphics[width=\textwidth]{piDStephens.ps}
\column{0.3\textwidth}
\raggedright
Observed Tajima's D {\small(Stephens et al 2003)}
\begin{itemize}
\pause
\item Too negative for multiregional hypothesis.\\
\pause
\item Not negative enough for replacement hypothesis.
\end{itemize}
\end{columns}
\end{frame}

\frame{\frametitle{Summary of Tajima's D}
Explains the pattern in the data better than any other
hypothesis.

\bigskip

\onslide<2>
If this hypothesis is correct, many human loci should have archaic
admixture.  How can we tell?
}

\begin{frame}
\frametitle{More disequilibrium in Europe than Africa}
\begin{columns}
\column{0.6\textwidth}
 \includegraphics[width=\textwidth]{ldcurve.ps}
\column{0.4\textwidth}
\raggedright
\begin{itemize}
\onslide<+->
\item More LD in Europe than Africa
\onslide<+->
\item Could reflect archaic admixture in Europe
\onslide<+->
\item or a European bottleneck.
\end{itemize}
\textsc{\footnotesize (Reich et al 2001)}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Decay of LD with distance along chromosome}
\includegraphics[width=\textwidth]{Jacobsson-LDdecay.eps}
\end{frame}

\begin{frame}
\frametitle{LD increases with distance from Africa}
\includegraphics[height=0.6\textheight,width=\textwidth]{Jacobsson-LDdist.eps}
\end{frame}

\frame{\frametitle{Disequilibrium at locus RRM2P4}
\includegraphics[width=\textwidth]{rrm2p4.eps}\\
\textsc{\footnotesize (Garrigan et al 2004)}
}

\begin{frame}
\frametitle{RRM2P4 continued}
\begin{columns}
\column{0.6\textwidth}
\includegraphics[width=\textwidth]{rrm2p4.eps}\\
\textsc{\footnotesize (Garrigan et al 2004)}
\column{0.4\textwidth}
\raggedright
\begin{itemize}
\item<alert@1> H'types A,B in strong disequilibrium
\item<alert@1> No surprise: sequence is short (2.4~kb)
\item<alert@2> Locus is very old: 2.5~myr
\end{itemize}
\end{columns}
\end{frame}

\begin{frame}
\frametitle{Deep disequilibrium at locus Xp21.1}
\begin{columns}
\column{0.6\textwidth}
\footnotesize
\renewcommand{\arraystretch}{0.7}
\begin{tabular}{cc@{}c@{}c@{}c@{}c@{}c@{}cc@{}c@{}c@{}c@{}c@{}c@{}cc@{}c@{}c@{}c@{}c@{}c@{}c@{}c@{}c@{}c}
          &  & & & & & & &  & & & & & & & 1&1&1&1&1&1&1&1&1&1\\
          &  & & & &1&1&1& 8&8&9&9&9&9&9& 5&5&5&5&6&6&6&6&7&7\\
          & 1&2&4&7&0&2&3& 4&5&2&3&4&4&6& 4&4&5&6&0&4&6&6&3&3\\
          & 5&1&3&3&6&9&1& 9&0&7&1&0&9&3& 5&6&4&6&8&2&3&4&4&6\\
          & 5&7&0&8&9&0&6& 1&2&4&6&4&5&8& 2&3&8&0&0&6&5&1&8&5\\
\hline
    a     & .&.&.&t&.&.&a& c&.&a&.&.&.&.& .&a&a&.&.&.&.&a&.&.\\
    b     & .&c&.&t&.&.&.& .&.&a&.&.&.&.& c&a&a&.&.&.&.&a&.&.\\
    C     & .&.&G&.&T&C&.& .&G&.&.&.&.&G& .&.&.&.&T&.&.&.&T&C\\
    d     & .&.&.&t&.&.&.& .&.&a&.&.&.&.& .&a&a&.&.&.&.&a&.&.\\
    e     & .&.&.&t&.&.&.& c&.&a&.&.&.&.& .&a&a&.&.&.&.&a&t&c\\
    f     & .&.&.&t&.&.&.& .&.&a&.&.&.&.& .&a&a&.&.&.&.&a&t&c\\
    g     & c&.&.&t&.&.&.& .&.&a&.&.&.&g& .&a&a&.&.&.&g&a&t&c\\
    h     & c&.&.&t&.&.&.& .&.&a&.&t&g&.& .&a&a&.&.&.&g&a&t&c\\
    i     & c&.&.&t&.&.&.& .&.&a&.&.&g&.& .&a&a&.&.&.&g&a&t&c\\
    j     & c&.&.&t&.&.&.& .&.&a&.&.&.&.& .&a&a&.&.&.&g&a&t&c\\
    k     & c&.&.&t&.&.&.& .&.&a&.&.&g&.& .&a&a&.&.&.&.&a&t&c\\
    l     & c&.&.&t&.&.&.& .&.&a&.&.&g&.& .&a&a&g&.&.&g&a&t&c\\
    m     & .&.&.&t&.&.&a& c&.&a&.&.&.&.& .&a&a&.&.&c&.&a&.&.\\
    n     & .&.&.&t&.&.&.& c&.&a&g&.&.&.& .&a&a&.&.&.&.&a&.&.\\
    o     & .&.&.&t&.&.&.& .&.&a&g&.&.&.& .&a&a&.&.&.&.&a&.&.\\
    p     & .&.&.&t&.&.&.& c&.&a&.&.&.&.& .&a&a&.&.&.&.&a&.&c\\
    q     & c&.&.&t&.&.&.& .&.&a&g&.&g&.& .&a&a&.&.&.&g&a&t&c\\
\end{tabular}%
\column{0.4\textwidth}
\raggedright
\begin{itemize}
\item Sequence is 17.5~kb long
\pause
\item MRCA is 1.9~myr old
\pause
\item H'type C in strong disequilibrium
\pause
\item Not balancing selection: too much disequilibrium
\pause
\item Requires isolated populations

\textsc{(Garrigan et al 2005)}
\end{itemize}
\end{columns}
\end{frame}

\frame{\frametitle{Microcephalin (Evans et al 2006)}
\begin{itemize}
\item Sequenced a large (23.4~kb) region.
\pause
\item One haplotype has been increasing for 37~kyr.
\pause
\item In ``near-complete LD across the entire region.''
\pause
\item MRCA 1.7~mya
\pause
\item Can't be balancing selection (too much LD).
\pause
\item Suggests archaic admixture.
\end{itemize}
}

\frame{\frametitle{Current views}
\begin{itemize}
\item Everyone agrees that there was little or no admixture of neutral
  archaic alleles.
\item Neanderthal alleles are different at several nuclear
  protein-coding loci.
\item Authors disagree about the likelihood of adaptive introgression.
\item We still don't know.
\end{itemize}
}

\frame{\frametitle{Update: 2010} Neandertal genome (Green et al 2010)
  indicates that non-African populations have some Neandertal
  ancestry.

\centering
\begin{tabular}{lcc}
           & Neandertal\\
Population & ancestry (\%)&S.E.\\ \hline
French     & 1.7 & 0.3\\
Han Chinese & 2.0 & 0.3\\
Papuan     & 1.5 &0.3
\end{tabular}\\

\pause
\begin{itemize}
\item Equal Neandertal admixture outside Africa
\item None within Africa
\end{itemize}
}

\frame{\frametitle{Summary}
\begin{itemize}
\item Hypothesis: our ancestors expanded out of Africa 50~kya,
and mated with surrounding archaic populations.
\pause
\item Yet only a few archaic genes entered the modern population.
\pause
\item Explains the contradictory signals given by different
  genetic loci.
\pause
\item It also predicts that some nuclear genes should show signs of
  archaic admixture: deep gene trees and high LD.
\pause
\item We find this pattern at two loci: Xp21.1 and Microcephalin.
\pause
\item Are we part Neandertal?  The jury is still out.
\end{itemize}
}

\end{document}

