% -*-latex-*-
\documentclass[handout]{beamer}
%\documentclass[]{beamer} 
\usepackage{etex}

\let\latexput\put
\usepackage{pictex}
\let\pictexput\put
\let\put\latexput

\newcommand{\G}{\mathcal{G}}
\newcommand{\Het}{\mathcal{H}}
\newcommand{\mutation}{$\cdot$} 
\newcommand{\OR}{\;\hbox{or}\;}
\newcommand{\AND}{\;\&\;}
\title{DNA Sequence Variation}
\author{Alan R. Rogers}
\date{\today}
\begin{document}

\frame{\titlepage}

\begin{frame}[containsverbatim]
\footnotesize
\begin{verbatim}
             0000000001 1111111112 2222222223 3333333334
             1234567890 1234567890 1234567890 1234567890

Sequence01   AATATGGCAC CTCCCAACCC TCTAGCATAT ACCACTTACA
Sequence02   .......T.. .C......TG C......C.. ..........
Sequence03   ..C....... .......... .......... ..........
Sequence04   .......T.. .C......TG C......... G.........
Sequence05   .......... .......... .......... ..........
Sequence06   .....A.... ........T. C......... G....C....
Sequence07   ..C....T.. .C......TG C......... G.........
Sequence08   .....A.T.. TC......TG C......... G.........
Sequence09   .......... .......... C......... ..........
Sequence10   .G...A.... ........T. C......C.. .T....C..G
Segregating:  ^^  ^ ^   ^^      ^^ ^      ^   ^^   ^^  ^
\end{verbatim}
\begin{center}
15 segregating sites
\end{center}
\end{frame}

\begin{frame}
  \frametitle{Measures of variation among DNA sequences}
  \begin{description}
    \item[Gene diversity per sequence] (a.k.a.\ heterozygosity)
  The probability that two random sequences differ.
    \pause
  \item[Number, $S$, of segregating sites] A ``segregating site'' is
    one that is polymorphic in the data.
    \pause
  \item[Mean pairwise difference, $\Pi$, per sequence] The average
    number nucleotide site differences between pairs of sequences. 
    \pause
  \item[Mean pairwise difference, $\pi$, per nucleotide] Equals $\pi =
    \Pi / L$, where $L$ is sequence length.
    \pause
  \item[Mismatch distribution] A histogram whose $i$th entry is the
   number of pairs of sequences that differ by $i$ sites.
    \pause
  \item[Site frequency spectrum] A histogram whose $i$th entry is the
  number of polymorphic sites at which the mutant allele is present in
  $i$ copies within the sample.
  \end{description}
\end{frame}

\begin{frame}
  \frametitle{The number, $\binom{k}{2}$, of ways to choose 2 items
    from $k$}

There are $k$ ways to choose the first item.  Having chosen the first,
there are $k-1$ ways to choose the second, so there are $k(k-1)$
pairs.  But this counts pair $AB$ separately from $BA$.  We are
interested in unordered pairs, so
  \[
  \binom{k}{2} = k(k-1)/2
  \]
\end{frame}

\begin{frame}[containsverbatim]
\frametitle{A a set of made-up DNA sequences}
\begin{verbatim}
         00000 00001
         12345 67890
S1       AAACT GTCAT
S2       ..... A....
S3       ..... A...C
S4       ..G.. A....
S5       ..G.. A....
\end{verbatim}

\bigskip

Calculate the mean pairwise difference, the number of segregating
sites, the mismatch distribution and the site frequency spectrum.
\end{frame}

\begin{frame}[containsverbatim]
\frametitle{Mean pairwise difference (MPD)}
\begin{verbatim}
   00000 00001       Pair Diff   Pair Diff
   12345 67890      (1,2)   1   (2,5)   1
S1 AAACT GTCAT      (1,3)   2   (3,4)   2
S2 ..... A....      (1,4)   2   (3,5)   2
S3 ..... A...C      (1,5)   2   (4,5)   0
S4 ..G.. A....      (2,3)   1   Sum diffs: 14 
S5 ..G.. A....      (2,4)   1   MPD/seq  : 14/10
\end{verbatim}

\bigskip
\begin{columns}
\column{0.35\textwidth}
\begin{tabbing}
Columnx\= \kill
Column \> Differences\\
03 \> $2\times 3$\\
06 \> $1\times 4$\\
10 \> $1\times 4$\\
Sum \> 14
\end{tabbing}
\column{0.65\textwidth}
\noindent
Number of pairs: $(5 \times 4)/2 = 10$

\medskip
MPD per sequence: $\Pi = 14/10$

\medskip
MPD per site: $\pi = 14/(10\times 10)$
\end{columns}
\end{frame}

\begin{frame}[containsverbatim]
\frametitle{Mismatch distribution}
\begin{verbatim}
   00000 00001       Pair Diff   Pair Diff
   12345 67890      (1,2)   1   (2,5)   1
S1 AAACT GTCAT      (1,3)   2   (3,4)   2
S2 ..... A....      (1,4)   2   (3,5)   2
S3 ..... A...C      (1,5)   2   (4,5)   0
S4 ..G.. A....      (2,3)   1
S5 ..G.. A....      (2,4)   1
\end{verbatim}

\bigskip
\begin{center}
Mismatch distribution\\[1ex]
\begin{tabular}{lrrr}
Differences & 0 & 1 & 2\\
Count       & 1 & 4 & 5\\
\end{tabular}
\end{center}
\end{frame}

\begin{frame}
  \frametitle{Calling ancestral and derived alleles}
  {\centering\input{figancder}\\[2ex]}

  \begin{itemize}
  \item Two hypotheses about which allele is ancestral.
  \item ``C'' requires 2 mutations; ``T'' requires 1.
  \item Because mutations are rare, ``T'' is more likely.
    \item When the in-group is polymorphic, the ancestral allele is
      usually the one present in the out-group.
  \end{itemize}
\end{frame}

\begin{frame}[containsverbatim]
\frametitle{Unfolded site frequency spectrum}
\begin{columns}
\column{0.4\textwidth}
\begin{verbatim}
       123456
Human1 AATAGC
Human2 ..AC..
Human3 .TACT.
Human4 ..ACT.
-------------
Chimp  AAAATC
\end{verbatim}

\vfill

\mbox{}\\
\column{0.6\textwidth}
1: fixed.

2: T derived; singleton.

3: T derived; singleton.

4: C derived; tripleton.

5: G derived; doubleton.

6: fixed.

\hrulefill\\

\begin{tabular}{lr}
Singletons & 2\\
Doubletons & 1\\
Tripletons & 1
\end{tabular}
\end{columns}
\end{frame}
\end{document}
